content/en/serverless/installation/go.md

```diff
@@ -78,9 +78,42 @@ datadog-ci lambda instrument -f my-function -f another-function -r us-east-1 -e
 
 More information and additional parameters can be found in the [Datadog Serverless CLI][2].
 
+
 [1]: https://aws.github.io/aws-sdk-go-v2/docs/getting-started/#get-your-aws-access-keys
 [2]: https://docs.datadoghq.com/serverless/serverless_integrations/cli
 {{% /tab %}}
+{{% tab "Serverless Framework" %}}
+
+The [Datadog Serverless Plugin][1] automatically adds the Datadog Lambda Library to your functions using Lambda Layers, and configures your functions to send metrics, traces, and logs to Datadog through the [Datadog Lambda Extension][2].
```

Comment:
Likewise the plugin doesn't add the Lambda Library for Go -- the customer must add the Lambda Library themselves directly to the code.